Makefile

```diff
@@ -115,9 +115,9 @@ lint: lint-static-check
 
 .PHONY: install-tools
 install-tools:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint
-	go install github.com/jstemmer/go-junit-report
-	go install honnef.co/go/tools/cmd/staticcheck
+	GOBIN=$(PWD)/bin go install github.com/golangci/golangci-lint/cmd/golangci-lint
+	GOBIN=$(PWD)/bin go install github.com/jstemmer/go-junit-report
```

Comment:
where's go-junit-report called?

Reply:
I pulled it for `go test` but it is not used anymore since we changed to `codecov` for test coverage result visualization later. removed it. Thanks.